gsy_framework/schema/avro_schemas/simulation_raw_data_v1.json

```diff
@@ -0,0 +1,562 @@
+{
```

Comment:
I did not understand for which data this schema is for (from the file name). I think that "raw_data" is a too broad expression, but I can also not think of a better name. Nothing to do here, only if you can think of another word for it. Thanks!

Reply:
I borrowed this name from [here](https://github.com/gridsingularity/gsy-e/blob/3c5303ace1d0b46b3a7b2aadec4d8b84d8890cf6/src/gsy_e/gsy_e_core/sim_results/endpoint_buffer.py#L158). And I'm open to any good name you suggest.